web/app/hooks/useBuildServiceWorker.ts

```diff
@@ -0,0 +1,40 @@
+import { useLocation, useMatches } from 'remix';
+import { useEffect } from 'react';
+
+let isMount = true;
+
+export function useBuildServiceWorker() {
+  const matches = useMatches();
+  const location = useLocation();
+
+  useEffect(() => {
+    let mounted = isMount;
```

Comment:
what is the reason to use let instead of const?

Reply:
This one is reassigned!